lib/config/options/index.ts

```diff
@@ -1576,16 +1576,15 @@ const options: RenovateOptions[] = [
     supportedPlatforms: ['azure', 'gitea', 'github', 'gitlab'],
   },
   {
-    name: 'stabilityDays',
-    description:
-      'Number of days required before a new release is considered stable.',
-    type: 'integer',
-    default: 0,
+    name: 'minimumReleaseAge',
+    description: 'Time required before a new release is considered stable.',
+    type: 'string',
+    default: '0',
```

Comment:
```suggestion
```

default should be `null`

Reply:
I created an issue for us to use `auto/disabled/enabled` instead of `null`. So it surprises me that you want to use `default=null`. Can you review this issue, and tell me what you think?

- https://github.com/renovatebot/renovate/issues/9100

Reply:
this isn't an enum like option. so null is ok here.